docs/topics/mta-web-controlling-tasks-with-task-manager.adoc

```diff
@@ -0,0 +1,14 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-08-07
+:_mod-docs-content-type: PROCEDURE
+
+[id="mta-web-controlling-tasks-with-task-manager_{context}"]
+= Controlling {ProductShortName} tasks by using Task Manager
+
+*Task Manager* provides precise information about the {ProductFullName} tasks queued for execution. *Task Manager* handles the following types of tasks:
+
+* Application analysis
+* Language discovery
+* Technology discovery
+
+You can display task-related information by using either the *Task Manager* drawer for active tasks, or the *Task Manager* page in the *Migration* view for all tasks.
```

Comment:
@mpershina Please mention how to open the drawer "by clicking the notifications badge/button"

Reply:
Done, @mguetta1, see my suggestion below :-)

Reply:
LGTM